packages/hooks/src/useCheck10kLimit.ts

```diff
@@ -0,0 +1,21 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import { getMinWithdrawal } from '../../shared/src/utils/currency';
+
+type TUseCheck10kLimit = {
+    is_10k_withdrawal_limit_reached: boolean;
+    max_withdraw_amount: number | undefined;
+    isSuccess: boolean;
+};
+
+const useCheck10kLimit = (): TUseCheck10kLimit => {
+    const { client } = useStore();
+    const { data: account_limit, isSuccess } = useFetch('get_limits');
+    const remainder = account_limit?.get_limits?.remainder;
+    const min_withdrawal = getMinWithdrawal(client.currency);
```

Comment:
Add [these](https://github.com/binary-com/deriv-app/pull/8867/files#diff-f7f1ad92a942db04a6a3c1953f106bcee105f151472f6098ec5795b534b2d8d1) changes to the `useCurrencyConfig` hook and then use `useCurrentCurrencyConfig` instead of this method 🙏🏻